state/pruning_state.py

```diff
@@ -106,9 +106,11 @@ def generate_state_proof(self, key: bytes, root=None, serialize=False, get_value
         return self._trie.generate_state_proof(key, root, serialize, get_value=get_value)
 
     def generate_state_proof_for_keys_with_prefix(self, key_prfx, root=None,
-                                                  serialize=False, get_value=False):
+                                                  serialize=False, get_value=False,
+                                                  from_seq_no=None, limit=None):
```

Comment:
This is a general state trie method where the key can be anything, not necessary `seq_no`.
So, `from_seq_no` argument name doesn't make sense here.